### ISSUE TITLE
More icon sizes

### ISSUE THREAD

**Issue by jlu5** [GLolol/gnome-exe-thumbnailer#3]
G-e-t icons are currently very pixelated on Nautilus - we should consider generating more icon sizes, especially for programs that provide them.

![get-scaling](https://user-images.githubusercontent.com/4644601/28096962-8beaa65a-6661-11e7-8fdd-1b9324338d85.png)

**Comment by jlu5** [comment]
It looks like doing this would also require reworking all the installer and template resources, which are currently hardcoded to 32x32 and 48x48 respectively.

**Comment by jlu5** [comment]
The successor to this project, [icoextract](https://github.com/exe-thumbnailer/exe-thumbnailer/issues/17), includes native support for 256x256 icons.

**Comment by MangaD** [comment]
Will it be possible to use icoextract with thunar? Is it usable right now?

**Comment by jlu5** [comment]
Yes, you can install it from Git right now following the instructions at https://github.com/jlu5/icoextract#installation. Make sure you uninstall exe-thumbnailer first, since the two try to thumbnail the same file types.

**Comment by MangaD** [comment]
I followed the instructions. I'm using XFCE and thunar but the icons didn't show, could it be due to large file sizes? Or should I edit some configuration (tumblerd?)?

**Comment by jlu5** [comment]
Is tumbler installed? Maybe try restarting thunar (`thunar --quit`)

**Comment by MangaD** [comment]
Just restarted PC, works fantastic!
